EnglishTranslation-master/SKILL.tsv

```diff
@@ -4129,7 +4129,7 @@ SKILL_20150714_004128	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRati
 SKILL_20150714_004129	Removes the armor's effect from the target.
 SKILL_20150714_004130	Decreases the target's defense.
 SKILL_20150714_004131	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}AoE Attack Ratio: #{SkillSR}#
-SKILL_20150714_004132	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Ignores the defense of small-type and medium-type targets #{CaptionRatio}#{nl}AoE Attack Ratio: #{SkillSR}#
+SKILL_20150714_004132	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Defense Ignored: #{CaptionRatio}#{nl}AoE Attack Ratio: #{SkillSR}#
```

Comment:
You removed the part where it says it ignores the defense of **small and medium-sized/type targets**.